__tests__/artifacts.test.js

```diff
@@ -146,7 +150,9 @@ describe.each(browsers)('Generated list artifacts (%s)', browser => {
       indexPath = path.resolve(result.resultsPath, '..', 'index.html');
       expect(fs.existsSync(indexPath)).toBe(true);
     } finally {
-      cleanup([path.resolve(result?.resultsPath), indexPath]);
+      if(!process.env.CI){
```

Comment:
This we need in all environment, not just CI to make sure that when tested locally, each browser run regenerates the `results/index.html` every time.